model/articlesModel.js

```diff
@@ -29,5 +29,57 @@ exports.selectAllArticles = () =>{
     })
 }
 
+exports.SelectPatchedArticleById = (voteQuery,id) =>{
+
+    const checkIdExists = db.query(`
+    SELECT EXISTS(SELECT 1 FROM articles WHERE article_id = $1)
+    `,[id])
+
+    return checkIdExists
+    .then((existsCheckId)=>{
+
+        if(existsCheckId.rows[0].exists){
+            if(voteQuery >= 0){
```

Comment:
Suggestion: the simpler way of doing this is to just update the query with votes + inc_votes, regardless of whether it is plus or minus:
100 + 1 = 101
100 + -1 = 99
it works for incrementing and decrementing the votes without the need for any additional logic or extracted functions - we can do the query directly in the model